### PR TITLE
WatchGuardFirebox Connector update

### PR DESCRIPTION
**Pull request by ecosystempo**
Fixes #

New update WatchGuardFirebox Connector and related files

Connector_syslog_WatchGuardFirebox.json
WatchGuard_Logo-SVG_Format.svg
WatchGuardFirebox.txt
WatchGuardFirebox_syslog_data.csv 

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Review by v-jayakal — CHANGES_REQUESTED** [review]
Can you please correct the issues with Logo?
Ensure raw file of logo does not have xmlns:xlink.
id should have guid. If some properties in the logo have IDs (for e.g. <g id="layer0"...), then set these IDs as GUIDs so that these are uniquely identifiable across all Azure logo assets

Refer https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md for logo guidelines.

File: DataConnectors/Connector_syslog_WatchGuardFirebox.json [hunk 1]
```diff
@@ -0,0 +1,118 @@
+{
+    "id": "WatchguardFirebox",
+    "title": "Watchguard Firebox",
+    "publisher": "WatchGuard Technologies",
+    "descriptionMarkdown": "Watchguard Firebox （https://www.watchguard.com/wgrd-products/firewall-appliances and https://www.watchguard.com/wgrd-products/cloud-and-virtual-firewalls） is security products/firewall-appliances. Watchguard Firebox will send syslog to Watchguard Firebox collector agent.The agent then sends the message to the workspace.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **WatchguardFirebox** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinelgithubparsersWatchguardFirebox)",
```
Comment (v-jayakal) [inline DataConnectors/Connector_syslog_WatchGuardFirebox.json:6]: Please use **https://aka.ms/sentinel-watchguardfirebox-parser** as Kusto Function link.

**Comment by ecosystempo** [comment]
Update WatchGuardFirebox.txt and Connector_syslog_WatchGuardFirebox.json file following the request, please have a check
As for WatchGuard_Logo-SVG_Format.svg file
I remove xmlns:xlink, keep id="Layer_1".
logo properties didn't have GUIDs so i'm not sure id="Layer_1" should be keep or modify or delete
Thanks very much

File: DataConnectors/Connector_syslog_WatchGuardFirebox.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                    "write": true,
+                    "delete": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "", 
+            "description": ">This data connector depends on a parser based on a Kusto Function to work as expected. [Follow these steps](https://aka.ms/sentinelgithubparsersWatchguardFirebox) to create the Kusto Functions alias, **WatchguardFirebox**",
```
Comment (v-jayakal) [inline DataConnectors/Connector_syslog_WatchGuardFirebox.json:63]: Please update the Kusto Function Path here: https://aka.ms/sentinel-watchguardfirebox-parser

**Review by v-jayakal — CHANGES_REQUESTED** [review]
Logo should not have id. There is an invalid attribute version. 
Removed id and version and checked at my end, it looks good. Can you please check the attached file and make changes to the svg file if you find it good?

[WatchGuard_Logo-SVG_Format-updated.zip](https://github.com/Azure/Azure-Sentinel/files/5634924/WatchGuard_Logo-SVG_Format-updated.zip)

**Comment by ecosystempo** [comment]
Hi @v-jayakal 
Kusto Function Path I have already update yesterday, please have a check
image

**Comment by ecosystempo** [comment]
Hi @v-jayakal the logo file in attached  looks good, i did the update

**Comment by ecosystempo** [comment]
That file modification still has the xmlns entry. Attached is the file that was modified to be provided. This formatting should meet the requirements of:

https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md

Regards,


John Berg | Technical Program Manager
WatchGuard Technologies, Inc. | www.watchguard.com<http://www.watchguard.com/>
john.berg@watchguard.com<mailto:john.berg@watchguard.com>
Office Phone: 206-613-6697
Office Hours: 7:30AM - 4:30PM (Pacific Time), Monday – Friday

**Review by v-jayakal — APPROVED** [review]

**Review by nazang — APPROVED** [review]

**Pull request merged**